vaadin-grid-pro-flow/src/main/java/com/vaadin/flow/component/gridpro/GridPro.java

```diff
@@ -108,6 +108,12 @@ public GridPro(int pageSize) {
         setup();
     }
 
+    @Override
+    public void onEnabledStateChanged(boolean enabled) {
+        super.onEnabledStateChanged(enabled);
+        getElement().executeJs("this._editingDisabled = $0;", !enabled);
```

Comment:
Why use `executeJs` for this? I assume the grid wouldn't have `_editingDisabled` on anymore after page refresh (or grid hide/unhide) if `executeJs` is used.

Reply:
Done.